docs/sources/reference/api/docker_remote_api_v1.18.md

```diff
@@ -212,56 +212,56 @@ Json Parameters:
-        should map to. It should be specified in the form
-        `{ <port>/<protocol>: [{ "HostPort": "<port>" }] }`
-        Take note that `port` is specified as a string and not an integer value.
-  -   **PublishAllPorts** - Allocates a random host port for all of a container's
-        exposed ports. Specified as a boolean value.
-  -   **Privileged** - Gives the container full access to the host.  Specified as
-        a boolean value.
-  -   **ReadonlyRootfs** - Mount the container's root filesystem as read only.
-        Specified as a boolean value.
-  -   **Dns** - A list of dns servers for the container to use.
-  -   **DnsSearch** - A list of DNS search domains
-  -   **ExtraHosts** - A list of hostnames/IP mappings to be added to the
-      container's `/etc/hosts` file. Specified in the form `["hostname:IP"]`.
-  -   **VolumesFrom** - A list of volumes to inherit from another container.
-        Specified in the form `<container name>[:<ro|rw>]`
-  -   **CapAdd** - A list of kernel capabilties to add to the container.
-  -   **Capdrop** - A list of kernel capabilties to drop from the container.
-  -   **RestartPolicy** – The behavior to apply when the container exits.  The
-          value is an object with a `Name` property of either `"always"` to
-          always restart or `"on-failure"` to restart only when the container
-          exit code is non-zero.  If `on-failure` is used, `MaximumRetryCount`
-          controls the number of times to retry before giving up.
-          The default is not to restart. (optional)
-          An ever increasing delay (double the previous delay, starting at 100mS)
-          is added before each restart to prevent flooding the server.
-  -   **NetworkMode** - Sets the networking mode for the container. Supported
-        values are: `bridge`, `host`, and `container:<name|id>`
-  -   **Devices** - A list of devices to add to the container specified in the
-        form
-        `{ "PathOnHost": "/dev/deviceName", "PathInContainer": "/dev/deviceName", "CgroupPermissions": "mrw"}`
-  -   **Ulimits** - A list of ulimits to be set in the container, specified as
-        `{ "Name": <name>, "Soft": <soft limit>, "Hard": <hard limit> }`, for example:
-        `Ulimits: { "Name": "nofile", "Soft": 1024, "Hard", 2048 }}`
-  -   **SecurityOpt**: A list of string values to customize labels for MLS
-      systems, such as SELinux.
-  -   **LogConfig** - Logging configuration to container, format
-        `{ "Type": "<driver_name>", "Config": {"key1": "val1"}}
-        Available types: `json-file`, `syslog`, `none`.
-        `json-file` logging driver.
-  -   **CgroupParent** - Path to cgroups under which the cgroup for the container will be created. If the path is not absolute, the path is considered to be relative to the cgroups path of the init process. Cgroups will be created if they do not already exist.
+    -   **Binds** – A list of volume bindings for this container. Each volume
```

Comment:
Note to reviewers; these whitespace changes look correct; this and below is a nested bullet-list for HostConfig